docs/src/pages/components/AriaLive/WebAriaLivePage.js

```diff
@@ -0,0 +1,281 @@
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/* @flow strict */
+
+import React from 'react';
+import BpkAriaLive from 'bpk-component-aria-live';
+import BpkChip from 'bpk-component-chip';
+import BpkSwitch from 'bpk-component-switch';
+import BpkLink from 'bpk-component-link';
+import BpkFieldset from 'bpk-component-fieldset';
+import { BpkCode } from 'bpk-component-code';
+import BpkSelect from 'bpk-component-select';
+import readme from 'bpk-component-aria-live/README.md';
+import { cssModules } from 'bpk-react-utils';
+
+import { WebComponentPage } from '../../../components/ComponentPage';
+import Paragraph from '../../../components/Paragraph';
+
+import STYLES from './aria-live-page.scss';
+
+const getClassName = cssModules(STYLES);
+
+class SelectExample extends React.Component<
+  {},
+  { destination: string, direct: boolean },
+> {
+  constructor() {
+    super();
+    this.state = {
+      destination: 'Panjin',
+      direct: false,
+    };
+  }
+
+  onChangeDestination = destination => {
+    this.setState({ destination });
+  };
+
+  toggleDirectness = () => {
+    this.setState(prevState => ({ direct: !prevState.direct }));
+  };
+
+  id = 'aria-live-select-example';
+
+  render() {
+    const { destination, direct } = this.state;
+    return (
+      <div>
+        <div>
+          <Paragraph>
+            Interactive component with
+            <BpkCode>aria-controls=&quot;{this.id}&quot;</BpkCode> to link it to
+            the ARIA live region below with the same ID.
+          </Paragraph>
+          <div
+            className={getClassName('bpkdocs-aria-live-page__select-wrapper')}
+          >
+            <BpkFieldset
+              label="Destination"
+              className={getClassName('bpkdocs-aria-live-page__select')}
+            >
+              <BpkSelect
+                id="destination"
+                name="destination"
+                value={destination}
+                onChange={event => {
+                  this.onChangeDestination(event.target.value);
+                }}
+              >
+                <option value="Abisko">Abisko</option>
+                <option value="Nara">Nara</option>
+                <option value="Panjin">Panjin</option>
+              </BpkSelect>
+            </BpkFieldset>
+            <BpkSwitch
+              className={getClassName('bpkdocs-aria-live-page__switch')}
+              label="Direct flights only"
+              checked={direct}
+              onChange={this.toggleDirectness}
+            />
+          </div>
+        </div>
+        <BpkAriaLive
+          visible
+          id={this.id}
+          className={getClassName('bpkdocs-aria-live-page__aria-live')}
+        >
```

Comment:
Because we are just changing the content that's there, a screen-reader will only read out the change. eg "Direct" or "Nara"
We should use `aria-atomic="true" to tell it we want to read the whole thing when a change is made.

We should also explain this in the example blurb

Reply:
Ahh, thanks! I knew I'd make some sort of mistake like that, I'll update it and the blurbs to reflect this.